TF/src/TFParser.py

```diff
@@ -79,15 +84,56 @@ def parse_graph(self, file_path, model_name, category,
             # Loop to populate to_nodes and from_nodes
             for operation in graph.get_operations():
 
+                # If graph contains StatefulPartitionedCall operation,
+                # only extracting the operations and returning empty graph
                 if operation.node_def.op == "StatefulPartitionedCall":
-                    print("Graphs with operation 'StatefulPartitionedCall' are not supported for parsing")
-                    return None
+                    print(
+                        "Graphs with operation 'StatefulPartitionedCall' are " 
+                        "not fully supported for parsing, graph or tensor " 
+                        "information not supported, only operators will be "
+                        "loaded into database."
+                        )
+
+                    NODES_DISCARDED = [
+                        "Const", "VarHandleOp", "StatefulPartitionedCall", 
+                        "NoOp", "Identity"
+                        ] # List of operations to not be considered, not of semantic use.
+
+                    nodes.clear()
+
+                    # Looping over all ops in the graph
+                    for node_def in graph_def.node:
+                        op = node_def.op
+                        if op in NODES_DISCARDED or "VariableOp" in op:
```

Comment:
May be if we add "VariableOp" in NODES_DISCARDED, we get the same logic but simpler?

Reply:
Slight difference in logic, `"VariableOp" in op` checks if its a sub-string in op, while `op in NODES_DISCARDED` checks for the equality matches.